src/libraries/System.Reflection.MetadataLoadContext/src/System/Reflection/TypeLoading/RuntimeTypeInfo.BindingFlags.cs

```diff
@@ -156,7 +156,7 @@ protected sealed override PropertyInfo GetPropertyImpl(string name, BindingFlags
                 }
 
                 if ((bindingAttr & BindingFlags.ExactBinding) != 0)
-                    return System.DefaultBinder.ExactPropertyBinding(candidates.ToArray(), returnType, types, modifiers);
+                    return System.DefaultBinder.ExactPropertyBinding(candidates.ToArray(), returnType!, types, modifiers!);
```

Comment:
NIT: `returnType` and `modifiers` could be nullable for the method `System.DefaultBinder.ExactPropertyBinding( ... )`